src/data/nat/basic.lean

```diff
@@ -614,6 +614,18 @@ begin
   exact mul_le_mul_of_nonneg_left (nat.succ_le_of_lt h) dec_trivial,
 end
 
+lemma lt_mul_of_one_lt_left {m n : ℕ} (hm : 0 < m) (hn : 1 < n) : m < n * m :=
```

Comment:
Can we have this, e.g., for an `ordered_semiring`?

Reply:
It turns out we had `lt_mul_of_one_lt_right` (but not `_left`) for `linear_ordered_semiring` (though `ordered_semiring` is sufficient for this direction, as you suggested). Fixed.